pkg/apiserver/api_installer.go

```diff
@@ -104,9 +104,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	admit := a.group.Admit
 	context := a.group.Context
 
-	serverGroupVersion := a.group.GroupVersion
+	externalGroupVersion := a.group.GroupVersion
 	if a.group.ServerGroupVersion != nil {
-		serverGroupVersion = *a.group.ServerGroupVersion
+		externalGroupVersion = *a.group.ServerGroupVersion
```

Comment:
> I don't get whyexternalGroupVersion is a better name. To me, serverGroupVersion makes more sense since it's server picks to use version.

In usage below, you use it as the `externalVersion` that will be decoded to match the `internalVersion` of the object you're interpretting.  Otherwise, you're decoding to `serverVersion`  and converting that to `internalVersion`.  External versions are serialized and internal versions are not.  Which one would you expect a server to use?